src/lib/codemirror-editor.ts

```diff
@@ -127,6 +132,12 @@ export class CodeMirrorEditorElement extends LitElement {
       }),
       root: this.shadowRoot!,
     });
+    // EditorView writes a <style> directly into the given root on construction
+    // (unless adopted stylesheets are available in which case it uses that).
+    // But then lit renders and blows it away. So, we'll just snatch any new
+    // styles beefore this can happen, and then have lit put them back again.
+    this._capturedCodeMirrorStyles = Array.from(
+      this.shadowRoot!.querySelectorAll('style'));
```

Comment:
Do we know that `new EditorView` reuses existing styles so we don't build up an every growing list of styles here every time we create a view?

If we don't, it's ok to leave a TODO to check this for now.

Reply:
Confirmed it re-uses the same style and tested nothing weird happens. Added a comment.